check.ts

```diff
@@ -0,0 +1,140 @@
+import { Request, Response } from 'express';
+
+import ReviewService from '@/services/reviewService';
+import type { PullRequestWebhookEvent, PullRequestReviewWebhookEvent, IssueCommentWebhookEvent } from '@/types';
+import config from '@/utils/config';
+import logger from '@/utils/logger';
+
+class WebhookController {
+  private reviewService: ReviewService;
+
+  constructor() {
+    this.reviewService = new ReviewService();
+  }
+
+  async handlePullRequest(req: Request<{}, {}, PullRequestWebhookEvent>, res: Response): Promise<void> {
+    try {
+      const { action, number, pull_request, repository } = req.body;
+
+      logger.info('Received PR webhook', {
+        action,
+        number,
+        repo: repository?.full_name,
+      });
+
+      // Only process certain actions
+      const reviewActions = ['opened', 'synchronize', 'reopened'];
+      if (!reviewActions.includes(action)) {
+        logger.info('Ignoring PR action', { action });
+        return res.status(200).json({ message: 'Action ignored' });
+      }
+
+      // Only process PRs for configured repository
```

Comment:
The commented-out repository validation logic should either be implemented or removed. If this is meant for multi-repo support later, consider adding a TODO comment explaining the future plans. If not needed, clean up the code.

```suggestion
      // TODO: Add repository validation when multi-repo support is needed
      // const owner = repository.owner.login;
      // const repo = repository.name;
```